src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -2093,12 +2093,6 @@ public FixableDiagnostic ProviderDeclarationViaImportKeywordIsDeprecated(Provide
                     DiagnosticStyling.Default,
                     codeFix);
             }
-
```

Comment:
Removing this error message since its redundant with :
![image](https://github.com/Azure/bicep/assets/9611108/36fc3c33-7334-4361-8fda-10e7742b4c71)